tensorflow_probability/python/distributions/laplace.py

```diff
@@ -202,6 +202,11 @@ def _median(self):
   def _mode(self):
     return self._mean()
 
+  def _quantile(self, x):
+    loc = tf.convert_to_tensor(self.loc)
```

Comment:
You shouldn't use the `convert_to_tensor` here, using `self.loc` and `self.scale` directly should be fine here.

Reply:
Noted and will change.